### DISCUSSION TITLE
Deprecated Next Browser replaced with Nyxt Browser

### DISCUSSION THREAD

**Discussion by andrewcrook** [Homebrew/discussions#3417] [Casks]
Cask for Next Browser says that it is deprecated 

```
  caveats do
    discontinued
  end
```
However, since the browser seems to have changed name and has been updated as `Nyxt `

Cask
https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/next.rb

New 
[nyxt.atlas.engineer](https://nyxt.atlas.engineer/)

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask/pull/123958
